cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -276,4 +276,7 @@ if (${LLVM_PACKAGE_VERSION} VERSION_LESS "8.0")
 endif ()
 llvm_map_components_to_libnames(LLVM_LIBRARIES core mcjit nativecodegen native ipo)
 include_directories(SYSTEM ${LLVM_INCLUDE_DIRS})
-list(APPEND TERRIER_LINK_LIBS ${LLVM_LIBRARIES})
\ No newline at end of file
+list(APPEND TERRIER_LINK_LIBS ${LLVM_LIBRARIES})
+
+#flatbuffers
+include_directories(SYSTEM "${THIRDPARTY_DIR}/flatbuffers/include")
```

Comment:
You might want to add a final newline here.

Reply:
Thanks!